src/layers/core/path-layer/path-layer-vertex.glsl.js

```diff
@@ -175,5 +175,8 @@ void main() {
   pos = lineJoin(prevPosition, currPosition, nextPosition);
 
   gl_Position = project_to_clipspace(vec4(pos, 1.0));
+
+  // Set color to be rendered to picking fbo which is used to check for selection highlight.
```

Comment:
Nit a little clearer if we change:
`Set color to be rendered to picking fbo which is used for ... `
=> 
`Set color to be rendered to picking fbo(also used for...)`
Please update in all placed